.rhcicd/pr_check.sh

```diff
@@ -22,22 +22,37 @@ COMMON_BUILDER=https://raw.githubusercontent.com/RedHatInsights/insights-fronten
 # --------------------------------------------
 # Options that must be configured by app owner
 # --------------------------------------------
-IQE_PLUGINS="hms-content"
-IQE_MARKER_EXPRESSION="smoke"
-IQE_FILTER_EXPRESSION=""
+export IQE_PLUGINS="hms-content"
+export IQE_MARKER_EXPRESSION="smoke and ui"
+export IQE_FILTER_EXPRESSION=""
+export IQE_ENV="ephemeral"
+export IQE_SELENIUM="true"
+export IQE_CJI_TIMEOUT="30m"
+export DEPLOY_TIMEOUT="900"  # 15min
+export DEPLOY_FRONTENDS="true"
+
+# bootstrap bonfire and it's config
+CICD_URL=https://raw.githubusercontent.com/RedHatInsights/bonfire/master/cicd
+curl -s "$CICD_URL"/bootstrap.sh > .cicd_bootstrap.sh && source .cicd_boot
```

Comment:
missing the end of the filename here, got cut off at `.cicd_boot`, so it never ran.